### ISSUE TITLE
Help: celeros.de as ddns service

### ISSUE THREAD

**Issue by softwarecrash** [qdm12/ddns-updater#669]
hello,
i have domains by celeros.de and can assign ddns to subdomains.
but the provider only give me this example for updating the ip

`https://kc.celeros.de/dyndns?domain=<domain>&ip=<ipaddr>&ip6=<ip6addr>&key=<pass>`

manual when i call this adress only with domain and ip it will update.

**Comment by qdm12** [comment]
Note you can use https://github.com/qdm12/ddns-updater/blob/master/docs/custom.md for the time being

**Comment by softwarecrash** [comment]
ah great, why i doenst find that, thank you!

**Comment by softwarecrash** [comment]
> Note you can use https://github.com/qdm12/ddns-updater/blob/master/docs/custom.md for the time being

so now i have found the time to try this, but it create strange get url

this is the config i tried
```
{
  "settings": [
    {
      "provider": "custom",
      "domain": "my.dns.dev",
      "host": "@",
      "url": "https://kc.celeros.de/dyndns?domain=my.dns.dev&key=xxxxxxxxxxxxxxxxx",
      "ipv4key": "ip",
      "ipv6key": "ip6",
      "success_regex": "1",
      "ip_version": "ipv4",
      "ipv6_suffix": ""
    }
  ]
}
```

so in the debug logs from the updater it remove domain and replace it with hostname and remove the key

`DEBUG GET https://kc.celeros.de/dyndns?hostname=mydns.dev&ip=1.2.3.4 | headers: User-Agent: DDNS-Updater`

**Comment by qdm12** [comment]
Indeed, reading my own code, what on earth was this, did I write this drunk?!

This should be fixed in ae2bcd55c81caa69fd5a336e5fc4c5c5c615b712 so now it keeps all your query parameters and only overrides the ip key(s). Let me know if it works 😉 
I might do a v2.6.1 release, since this is a rather critical bug.

**Comment by softwarecrash** [comment]
> Indeed, reading my own code, what on earth was this, did I write this drunk?!
> 
> This should be fixed in [ae2bcd5](https://github.com/qdm12/ddns-updater/commit/ae2bcd55c81caa69fd5a336e5fc4c5c5c615b712) so now it keeps all your query parameters and only overrides the ip key(s). Let me know if it works 😉 I might do a v2.6.1 release, since this is a rather critical bug.

thank you for the quick replay, so badly i use it as docker compose and dont know how to insert the patch in the container

**Comment by qdm12** [comment]
Just `docker-compose pull && docker-compose up -d`. The changes are automatically built and pushed to Docker hub as the image `qmcgaw/ddns-updater`

**Comment by softwarecrash** [comment]
> Just `docker-compose pull && docker-compose up -d`. The changes are automatically built and pushed to Docker hub as the image `qmcgaw/ddns-updater`

ah tnaks, i dont know that changes are directly push to the docker image.


works perfect, thank you very much!
```
  "settings": [
    {
      "provider": "custom",
      "domain": "my.domain.de",
      "host": "@",
      "url": "https://kc.celeros.de/dyndns?domain=my.domain.de&key=xxxxxxxxxxxx",
      "ipv4key": "ip",
      "ipv6key": "ip6",
      "success_regex": "1",
      "ip_version": "ipv4",
      "ipv6_suffix": ""
    }
  ]
}
```

debug output
DEBUG GET https://kc.celeros.de/dyndns?domain=my.domain.de&ip=1.2.3.4&key=xxxxxxxxxxxxxxxx
